src/client/Python/msrest/msrest/serialization.py

```diff
@@ -603,6 +603,8 @@ def serialize_date(attr, **kwargs):
         :param Date attr: Object to be serialized.
```

Comment:
@lmazuel - I've made a couple of minor changes to serialization.py, if you could do a quick review :)
They should not be breaking, and I doubt are used by the existing clients so may not need a new release yet.

Effectively it's a change to support default/constant date and datetime values by allowing strings to passed in instead of date or datetime objects. The strings would need to already be in the correct ISO formatting that the request will serialize to. Let me know if you think this is okay.